django/template/base.py

```diff
@@ -393,28 +393,37 @@ def create_token(self, token_string, position, lineno, in_tag):
 
 
 class DebugLexer(Lexer):
+    def _tag_re_split_positions(self):
+        last = 0
+        for match in tag_re.finditer(self.template_string):
+            start, end = match.span()
+            yield (last, start)
+            yield (start, end)
+            last = end
+        yield (last, len(self.template_string))
+
+    # This parallels the use of tag_re.split() in Lexer.tokenize().
+    def _tag_re_split(self):
+        template_string = self.template_string
+        for position in self._tag_re_split_positions():
+            yield template_string[slice(*position)], position
```

Comment:
IMO, we can use `self.template_string` without an extra variable.

Reply:
Sounds good.